synapse/events/__init__.py

```diff
@@ -199,7 +199,8 @@ def __init__(self, internal_metadata_dict: JsonDict):
     proactively_send: DictProperty[bool] = DictProperty("proactively_send")
     redacted: DictProperty[bool] = DictProperty("redacted")
     txn_id: DictProperty[str] = DictProperty("txn_id")
-    token_id: DictProperty[int] = DictProperty("token_id")
+    device_id: DictProperty[str] = DictProperty("device_id")
+    token_id: DictProperty[Optional[int]] = DefaultDictProperty("token_id", None)
```

Comment:
comments please. What do these fields mean? What does it mean for them to be absent, or None?